support/refc/datatypes.h

```diff
@@ -46,7 +46,7 @@ typedef struct
 typedef struct
 {
     Value_header header;
-    char payload[25];
+    char payload[0];
```

Comment:
Why stub `payload` out, rather than remove it completely?

Reply:
To somehow document that Value is a base struct and payload is these. But I can remove.

Reply:
I feel that documentation would go better in a comment, rather than a struct hack. We should only keep `payload` if we plan to use it somehow. Any uses I can think of would be better to cast to the appropriate type, first.

As an aside, since C99, the standard way to do the struct hack is with `[]`, rather than `[0]`.

Reply:
OK, removed the field, replaced with comment.